File: gex/Models/MapStats/MapStatsDailyOpeningLab.cs
using Dapper.ColumnMapper;
using gex.Code;
using System;

namespace gex.Models.MapStats {

    [DapperColumnsMapped]
    public class MapStatsDailyOpeningLab {

        [ColumnMapping("map_filename")]
        public string MapFilename { get; set; } = "";

        [ColumnMapping("gamemode")]
        public byte Gamemode { get; set; }

        [ColumnMapping("day")]
        public DateTime Day { get; set; }

        [ColumnMapping("definition_name")]
        public string DefinitionName { get; set; } = "";

        [ColumnMapping("count")]
        public int Count { get; set; }

        [ColumnMapping("wins")]
        public int Wins { get; set; }

        [ColumnMapping("timestamp")]
        public DateTime Timestamp { get; set; }

    }
}

File: gex/src/model/map_stats/MapStatsDailyOpeningLab.ts

export class MapStatsDailyOpeningLab {
    public mapFilename: string = "";
    public gamemode: number = 0;
    public day: Date = new Date();
    public definitionName: string = "";

    public count: number = 0;
    public wins: number = 0;

    public timestamp: Date = new Date();

    public static parse(elem: any): MapStatsDailyOpeningLab {
        return {
            ...elem,
            day: new Date(elem.day),
            timestamp: new Date(elem.timestamp)
        };
    }
}